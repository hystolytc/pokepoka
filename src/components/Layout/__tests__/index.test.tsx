import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { Layout } from '../index'

const describeComponent = '<Layout /> test case'

describe(describeComponent, () => {
  const { container } = render(
    <Layout>
      <p>Layout Test</p>
    </Layout>
  )

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})