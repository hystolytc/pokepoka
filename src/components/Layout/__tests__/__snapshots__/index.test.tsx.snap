// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> test case Render <Layout /> test case correctly 1`] = `
<div>
  <div
    class="css-kx8ax7"
  >
    <p>
      Layout Test
    </p>
  </div>
</div>
`;
